### PR TITLE
Windows Driver manufacturer fix

### PR DESCRIPTION
**Pull request by harrisonhjones**
Fixed manufacturer string. Updated Spark Core Manufacturer to Particle.io

This was a joint effort by myself, @kennethlimcp, and @ScruffR

### CODE REVIEW

**Comment by m-mcgowan** [comment]
Not only did the manufacturer string change (Which is fine in itself) but also DEVICE_DESCRIPTION has changed to PROVIDER in the config. Can someone explain what that does/why it's necessary?

**Comment by kennethlimcp** [comment]
@m-mcgowan,

It all depends on what Particle IO wants the manufacturer name to be..

If we use `DEVICE_DESCRIPTION` for the "MANUFACTURER" field in Windows:
- Spark Core --> Spark Core
- Photon --> Photon

It should be either `Spark Devices` `Particle IO` or whatever the company wants to be known as.

This https://github.com/spark/firmware/pull/447/files#diff-84eb7381a951faef223f48e35068bf2aL88 may not be required if the Manufacturer for Spark Core is listed as `Spark Devices`

TLDR: The `[Manufacturer] ` is not taking the correct string

**Comment by ScruffR** [comment]
If you're refering to the change of `%DEVICE_DESCRIPTION%` into `%PROVIDER%` that was the whole solution, since the `%...%` notation is sort of a forward reference to the actual string that should be used as value descriptor under the [Manufacturer] section.
It's only a placeholder for the value which is given to `PROVIDER` further down the file, rather than the one of `DEVICE_DESCRIPTIONA`.

**Comment by m-mcgowan** [comment]
Don't we then need similar changes to the photon.inf file?

**Comment by kennethlimcp** [comment]
This fix changes both `.inf` file for `[Manufacturer]` to take `%PROVIDER%`

For `photon.inf`, --> https://github.com/harrisonhjones/firmware/blob/develop/misc/driver/windows/photon.inf#L88
already set to `Particle IO`

For `spark_core.inf`, `PROVIDER` is updated to `Particle IO`

**Comment by technobly** [comment]
@m-mcgowan @harrisonhjones @kennethlimcp @ScruffR Great collab on this guys!  FYI: Just confirmed we are doing business as just "Particle", no " IO" at the end.

**Comment by harrisonhjones** [comment]
Thanks @technobly, see #450
